### ISSUE TITLE
Compute zoning distribution metrics

### ISSUE THREAD

**Issue by futurechris** [hackoregon/emergency-response#24]

**Comment by kielejocain** [comment]
Blocked by #4

Edit: As the zoning data now has its own issue, this is blocked by #41

**Comment by rcallihan** [comment]
Is this referring to City of Portland zoning? If so, I can help get this data.

**Comment by kielejocain** [comment]
It is; if you know where to find it that would be awesome

**Comment by rcallihan** [comment]
Can someone further define the metrics we want here? Knowing will help me set up the data appropriately. I may need to aggregate Portland's zoning classes into simpler classes (also possibly Gresham's as well). Two things I imagine we're measuring, but if there's something else, let me know: 

- Calculating total area of each zoning class per FMA. This would helpful in profiling a given FMA. 
- Intersect each incident with a zoning class to understand more about the location of a given incident (e.g. what types of incidents happen in residential vs industrial?)
- Something else?

**Comment by kielejocain** [comment]
I don't know about others, but your first bullet point was what I mostly had in mind.  It might be nice to report to the responders of an FMA what their zoning distribution looks like.  I also quite like your second idea; that could be one of the featured 'cards' we make for presentation night.

I know almost nothing about zoning data, unless it hasn't changed since the original Sim City (SNES version).
